tests/js/server/live-reload-tests.js

```diff
@@ -0,0 +1,337 @@
+/*
+
+    Test for "live reloading" when templates are changed.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+fluid.setLogging(true);
+
+var gpii  = fluid.registerNamespace("gpii");
+
+require("../../../index");
```

Comment:
Done, throughout.  There were a bunch of variations on this, and plain `require` calls for `gpii-*` packages that I also fixed.